app/views/external_users/claims/misc_fees/_misc_fee_fields.html.haml

```diff
@@ -8,7 +8,7 @@
         %label.form-label
           = t('.fee_type')
         %a{id: "misc_fee_#{@misc_fee_count}_fee_type"}
-        = f.collection_select :fee_type_id, @claim.eligible_misc_fee_types, :id, :description, {include_blank: ''.html_safe }, {class: 'typeahead js-misc-fee-type js-fee-type', style: 'width:350px'}
+        = f.collection_select :fee_type_id, @claim.eligible_misc_fee_types, :id, :description, {include_blank: ''.html_safe }, {class: 'typeahead js-misc-fee-type js-fee-type', style: 'width:330px'}
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2288/files#r176058306)

Reply:
I am going to rebuild these views soon - there are some conflicting styles and it is a patch.